hack/testing/check-curator.sh

```diff
@@ -29,15 +40,20 @@ TEST_DIVIDER="------------------------------------------"
 # key=./key-fluentd
 # oc get secret logging-fluentd --template='{{.data.cert}}' | base64 -d > cert-fluentd
 # cert=./cert-fluentd
-fpod=`oc get pods | awk '/-deploy/ {next}; /-build/ {next}; /^logging-fluentd-.* Running / {print $1}'`
-
-# where "yesterday", "today", etc. are the dates in UTC in YYYY.mm.dd format
+fpod=`oc get pods -l component=fluentd | awk '/fluentd/ {print $1}'`
```

Comment:
This is not prettier but it *does* prevent relying on the default output template:

    oc get pods -l provider=openshift,component=fluentd --template '{{with index .items 0}}{{.metadata.name}}{{end}}'